wcivf/apps/elections/import_helpers.py

```diff
@@ -576,10 +576,16 @@ def check_for_ee_updates(self):
         print("checking for recently updated EE")
         for election_id in self.ee_helper.iter_recently_modified_election_ids():
             print(election_id)
-            if self.ee_helper.ee_cache[election_id]["group_type"]:
+            group_type = self.ee_helper.ee_cache[election_id]["group_type"]
+            if group_type in ["election", "organisation"] or self.is_pcc_or_gla(
+                group_type, election_id
+            ):
                 election = Election.objects.get(slug=election_id)
```

Comment:
I think this is going to fail when the election id is something like `local.2024-05-02` or  - i.e. the group type is an `election`. Because I don't think WCIVF has those types of ids.

Reply:
I'm also not sure how explanations assigned to things like `mayor.london.2024-05-02` which are 'PostElections` are going to work, but that might be a separate issue.

Reply:
It seems simpler not to rely on group_type and go in the direction (which I think is where you were headed late yest) of splitting the election_ids and basing the choice of Election vs PostElection on the the org_type such as `org_type = election_id.split('.')[0]`. What do you think?

Reply:
Or more simply, 
if the `election_id` starts with local (and has 3 parts after being split), get the related PostElection
else, get the related Election

Reply:
I think you probably do need to consider group type, because that tells you (in conjunction with what kind of election it is) whether it is an (WCIVF) `Election`, `PostElection` or neither. 

i.e. a local with `group_type` of  `election` is neither. `group_type` of organisation corresponds to an `Election`, `group_type` of `None` is a `PostElection`.
i.e. 
https://whocanivotefor.co.uk/elections/local.2024-05-02/ -> 404
https://whocanivotefor.co.uk/elections/local.exeter.2024-05-02/ -> list of ballots
https://whocanivotefor.co.uk/elections/local.exeter.alphington.2024-05-02/ -> single ballot

In the case of PCC I'm not sure how it works. 
https://whocanivotefor.co.uk/elections/pcc.west-midlands.2024-05-02/ -> single ballot
https://whocanivotefor.co.uk/elections/pcc.2024-05-02/ -> 404

I don't know if there is an WCIVF `Election` object that corresponds to PCC elections happening on 2024-05-02. Is it the case that there's a `PostElection` with no `Election` object, or do the `Election` objects and `PostElection` objects have the same ids? 

Tricky.

For the purposes of this PR I would break it down into the following steps. 

1) When there is an explainer in EE that is on something that exists in the WCIVF `Election` table (i.e. ids match) then update the description field. Check that this is working and picking up updates - i.e. when the explainer is modified in EE WCIVF should grab it, even though nothing will have changed in YNR.

2) Do something about EE explainers that are attached to EE elections with `group_type == 'election'`. I'm not sure what the right thing to do is here, it might be specific to the kind of election. So in the case of locals we probably want to attach the explainer on `local.2024-05-02` to every WCIVF `Election` with an id that starts with `local` and ends with `2024-05-02`. In the case of `mayor.2024-05-02`  I don't think that will be the case, because there are different explainers for different mayoral elections. I think enumerating the different cases in a way @pmk01 can have input is probably wise here.

Reply:
> > In the case of PCC I'm not sure how it works.
> https://whocanivotefor.co.uk/elections/pcc.west-midlands.2024-05-02/ -> single ballot
> https://whocanivotefor.co.uk/elections/pcc.2024-05-02/ -> 404
> 
> I don't know if there is an WCIVF Election object that corresponds to PCC elections happening on 2024-05-02. Is it the case that there's a PostElection with no Election object, or do the Election objects and PostElection objects have the same ids?
> 

Yes they do. It is tricky, esp since we don't know which fields have been updated in EE. Given that `import_metadata_from_ee` can take an election or a ballot argument, maybe we do both.

Reply:
After trying some matching, I realised that
`pcc` doesn't always have a `group_type` and it has both a ballot and an election in WCIVF. 

With [51c680c](https://github.com/DemocracyClub/WhoCanIVoteFor/pull/1807/commits/51c680c6230ec2a2cdf3cff916aa03a556dd791d), I've started by skipping `election_ids` that are a local with `group_type` of election, since it is neither an Election nor a PostElection in WCIVF. Then I've done simple matching of ids - I found this was more effective than splitting up `election_ids` and matching on `org_type` and/or date once we threw away what wasn't needed. But I've possibly oversimplified. I did test it and it has worked as expected on a `pcc` but need to do more testing today.

Reply:
Right so I think that's step one, if you can also make use of the field that records the last time EE was checked. 

The limitation of that approach is if the explainer for all local elections is added to the `local.2024-05-02` then this will miss it. But I think that should be step 2 :)